tests/codegen_testcases/yul/evm_builtin.sol

```diff
@@ -1,4 +1,4 @@
-// RUN: --target ewasm --emit cfg -Onone --no-cse
+// RUN: --target evm --emit cfg -Onone --no-cse
 
 contract Testing {
```

Comment:
We don't have code generation for EVM, so how is this file tested?

Reply:
We have codegen for evm (as in `src/codegen/..`), just emit.